static-site-plugin.js

```diff
@@ -26,10 +27,11 @@ module.exports = class {
       const { filename } = this
       const { path: outputPath } = outputOptions
       const { options } = compiler
-      const { assets: clientAssets, outputOptions: clientOutput } = options.clientCompilation || {}
-      const appAssets = Object.keys(clientAssets).map((asset) => (
-        clientOutput.publicPath + asset
-      ))
+      const { clientStats = {} } = options
+      const { compilation: clientCompilation = { toJson () { return {} } } } = clientStats
```

Comment:
This smells a bit like an over-use/abuse of de-structuring and default params. Mainly that it's a bit of work to follow exactly what's happening:

* clientStats.compilation is being destructured and renamed to the new local var clientCompilation. That part is fine but...
* The default value which is a mock object that returns an empty object when toJson is called on it, being created inline, obfuscates the whole thing and makes dumb programmers like me have to take 3-4 minutes to figure it out 😛 

....and then the next line throws away the clientCompilation var after destructuring its `outputOptions` property, which won't exist if that default value was used.

Maybe just kill the default value? I'm overthinking this, if there's always a compilation key on `clientStats` then it doesn't matter either way